hoot-core/src/main/cpp/hoot/core/io/ApiDb.cpp

```diff
@@ -384,4 +401,14 @@ QSqlQuery ApiDb::_execNoPrepare(const QString& sql) const
   return q;
 }
 
+long ApiDb::round(double x)
+{
+  return (long)(x + 0.5);
+}
+
+long ApiDb::round(double x, int precision)
+{
+  return (long)(floor(x * (10 * (precision - 1)) + 0.5) / (10 * (precision - 1)));
```

Comment:
I'm looking at this, and... I'm thinking it's wrong. I think 10 should be raised to the power of (precision-1), not simply multiplied by (precision-1). Like use pow(10, precision-1) or something.